server/set-testing-db-url.sh

```diff
@@ -0,0 +1,13 @@
+if [[ -z $TESTING_DATABASE_URL ]];
+    then db=$(cat .env | grep 'TESTING' | sed 's/.*="\(.*\)"/\1/');
+    if [[ -z $db ]]; then echo 'TESTING_DATABASE_URL not specified' && exit 1
+    fi
+    else db=$TESTING_DATABASE_URL;
+fi;
+
+# echo "database url script ran"
+
+# export DATABASE_URL="$db"
+# echo "DATABASE_URL=$db" >> $GITHUB_ENV
```

Comment:
Remove this.

Reply:
Also, according to how it's used we should rename the file to `get-testing-db-url.sh`, as it doesn't set anything, but just returns the value.